src/styles/editor-skin.scss

```diff
@@ -161,4 +163,8 @@
 	.block-editor-list-view-leaf.is-selected td {
 		background-color: var(--stk-skin-dark);
 	}
+
+	.ugb-global-settings-typography-control .components-base-control__label {
+		overflow: hidden;
+	}
```

Comment:
This feels like this should belong to the global typography component and not in the editor skin